src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -490,17 +490,16 @@ export class ChatService extends Disposable implements IChatService {
 			this.trace('initializeSession', `Initialize session ${model.sessionId}`);
 			model.startInitialize();
 
-			await this.extensionService.whenInstalledExtensionsRegistered();
```

Comment:
This is a bit yolo: I am now not not even awaiting `whenInstalledExtensionsRegistered` so that the entire default agent activation is encapsulated in `activateDefaultAgent`. I hope that is still fine...

Reply:
I think so?